src/Crate/DBAL/Platforms/CratePlatform.php

```diff
@@ -102,9 +102,9 @@ public function supportsSequences()
     }
 
     /**
-     * If we want to support Schemas, we need to implement 
+     * If we want to support Schemas, we need to implement
      * getListNamespacesSQL and getCreateSchemaSQL methods
-     * 
+     *
```

Comment:
as nothing besides formatting was changed inside this file, I'd prefer to revert the changes